Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -351,6 +350,12 @@
           min: 1
         reagent: Histamine
         amount: 4
+      - !type:GenericStatusEffect
+        conditions:
+        - !type:TotalDamage
+          min: 90
+        key: PressureImmunity
+        component: PressureImmunity
```

Comment:
I feel like pure immunity is too much. even if you are at 90hp.

I can imagine players now saving the medipen for themselves in case they get in a spaced environment instead of helping others. Along with that since epi can be made. You can mass produce a lot and keep injecting yourself for as much as you have epi on a jug.

This is just a suggestion. I'm a contributor so you are free to ignore this.

Reply:
Do you think it only making you pressure immune while in crit would be balanced? You would need someone else to reapply the epi to you, and you won't be able to do anything untill they put a suit on you or drag you into an unspaced area.